src/Synthesis/PhaseAmplitudeConverter.icl

```diff
@@ -18,21 +18,11 @@ sineTableSize = 2205
 
 generateLocal :: !Int !Wave !Frequency -> Real
 generateLocal _ Silence _ = 0.0
-generateLocal localIndex Sine freq = sampleOut
+generateLocal localIndex Sine freq = sin actualPhase
```

Comment:
This shouldn't have the same output and should sound kind of weird tbh.
`actualPhase` in the original code calculates the index of the wavetable that should be sampled from, and performs a 4 point interpolation in order to account for the inevitable non-integer indices.
This can be made to work, in accordance to what John said in his email, if line 25 is changed to:
`actualPhase = (((toReal localIndex))/cycleSize)`

Reply:
Yea, the code should be 
`actualPhase = (((toReal localIndex))/cycleSize) * 2.0 * PI`
cycleSize is equal to number of samples in one cyle, which is equivalent to 2PI.
And the localIndex is the current index which we are at.